api/graphql/makeModels.js

```diff
@@ -42,8 +42,19 @@ export default function makeModels (userId, isAdmin) {
     users: {
       typename: 'Person',
       model: User,
-      attributes: ['id', 'name', 'avatar_url', 'banner_url'],
-      relations: ['posts'],
+      attributes: [
+        'id',
+        'name',
+        'avatar_url',
+        'banner_url',
+        'bio',
+        'twitter_name',
+        'linkedin_url',
+        'facebook_url',
+        'url',
+        'location'
+      ],
+      relations: ['memberships', 'posts'],
```

Comment:
Actually that's not going to work either, GraphQL is erroring out... don't merge this yet :)